aggregator/apps/api/v1/docs/finder_data_structures.apibp

```diff
@@ -63,6 +63,10 @@
                 + `election_name`: `Cardiff local election` (string) - Friendly name for this ballot's parent election group
                 + `post_name`: `Pontprennau/Old St. Mellons` (string) - Name of the division or post the winner(s) of this election will represent
                 + `candidates_verified`: `false` (boolean) - True if the list of candidates for this election has been confirmed against the nomination papers for this ballot. If this property is False, the candidate list is provisional or unconfirmed.
+                + `voting_system` (object) - The voting system used in this election
+                    + `slug` (string) - One of `AMS`, `FPTP`, `PR-CL`, `sv`, `STV`
+                    + `name` (string) - The name of this voting system (e.g: "First-past-the-post")
+                + `seats_contested`: (number) – The number of candidate who will be elected
```

Comment:
Can you provide some sensible example values here. A string value with no examples defined defaults to empty string when rendered and a number defaults to 0:


![Screenshot at 2020-02-17 10-40-19](https://user-images.githubusercontent.com/6025893/74646764-5dcc1980-5172-11ea-8d78-a3f9faf748c2.png)


Also I have just noticed that on the election endpoint, we expose also a `uses_party_lists` boolean undet the `voting_system` key:

https://github.com/DemocracyClub/aggregator-api/blob/d73f5d899e6f9626f2beee8ac0f6c294db5a7105/aggregator/apps/api/v1/docs/elections_data_structures.apibp#L75-L78

It would be useful to do the same here, both for consistency and just for giving users useful fields.

I should really have spotted that when reviewing https://github.com/DemocracyClub/WhoCanIVoteFor/pull/522 but I didn't :(

Reply:
Thanks, done both of these

Reply:
f1acc72 adds the field to the _docs_ but its still not in the actual API output.

Reply:
Oh.. I see you're working on it:
https://github.com/DemocracyClub/WhoCanIVoteFor/commit/956d250a4552946a94aa7fba20b1eefdc25a348a

can you make sure it gets added to the test input fixtures and sandbox outputs